content/src/fieldConfig/config.json

```diff
@@ -21,6 +21,9 @@
     "requiredTasksSingular": "${numTasks} required task",
     "requiredTasksPlural": "${numTasks} required tasks"
   },
+  "skipToMainContent": {
```

Comment:
[dust] We are trying to actively remove data from the `config` file. Could this live elsewhere?

Reply:
I'm open to suggestions for this one.